### ISSUE TITLE
Suffix % or any symbol for 'toPattern' same like toMoney

### ISSUE THREAD

**Issue by zaidchauhan** [fernandofleury/vanilla-masker#111]
Hello,

'toPattern' is really great to define custom masking. It must have option to pass suffix symbols same like toMoney. The reason is, many masking like percentage one will need '%' symbol as suffix in entered value.

**Comment by fernandofleury** [comment]
I don't think this is common usage to be honest. Does other mask plugins has this possibility?
